File: easy/1_two_sum.py
from typing import List


class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        lest = []
        for i in range(len(nums)-1):
            for l in range(len(nums)):
                if nums[i] + nums[l] == target and l != i:
                    lest.append(nums.index(nums[i]))
                    nums[i] += 1
                    if nums.index(nums[l]) not in lest:
                        lest.append(nums.index(nums[l]))
                        return lest
                    else:
                        return [nums.index(nums[i]), nums.index(nums[i])+1]

File: easy/1_two_sum.js
var twoSum = function(nums, target) {
    var map = {};
    for(var i = 0 ; i < nums.length ; i++){
        var n = nums[i];

        if(map[target-n] >= 0){
            return [map[target-n],i]
        } 
        else {
            map[n] = i;  
        }
    }
};